src/modules/auth/login.controller.ts

```diff
@@ -20,16 +20,19 @@ import { SiweMessage, generateNonce } from 'siwe';
 import parseDuration from 'parse-duration';
 import { LoginGuard } from './login.guard';
 import { TokenService } from './token.service';
-import { CookiesServices } from './cookies.service';
 import { RoleService } from '../role/role.service';
 import { SiweReqPayload } from './siwe.dto';
 
 @ApiTags('Auth')
 @Controller({ version: '1' })
 export class LoginController {
+  // Using "private readonly" class members so that automated scan can be certain that cookie options are set appropriately
+  private readonly cookieIsHttpOnly = true;
+  private readonly cookieIsSecure = true;
+  private readonly cookieSameSite = 'none';
```

Comment:
@nichonien how about this approach? Does this strike a better balance between reuse and immutability?

Reply:
Looks good for now, we can merge this change. It would be good to understand (some time later) why the scan doesn't pick the `cookieOptions` being imported or how exactly the code scan works.